homeassistant/components/mqtt/strings.json

```diff
@@ -23,6 +23,16 @@
   },
   "config": {
     "step": {
+      "user": {
+        "description": "Please choose how you want to connect to the MQTT broker:",
+        "menu_options": {
+          "addon": "Install and configure the official {addon} add-on.",
```

Comment:
It can happen that the user already have the add-on installed. We should either check that before showing the menu, or make this text more vague. Checking before showing the menu is probably better, the option would then either be to connect to the broker provided by the add-on, or to install and configure (what you already implemented)

Reply:
Made the text some what more vague about set up of the addon.